src/consumer/__tests__/index.test.ts

```diff
@@ -17,26 +18,33 @@ interface Params {
 
 function generateConfig(params: Params): Config {
   return {
-    host: 'kafka:9092',
+    host: KAFKA_HOST_9092,
     consumer: {
       groupId: 'my-group-id',
       ...params
     }
   };
 }
 
+let topicCounter = 0;
+
+function getIncrementalId(): number {
```

Comment:
Other option 
`function* getIncrementalId():number {
  let topicCounter = 0;
  while (true) {
    topicCounter++;
    yield topicCounter;
  }
}`
The advantage is it has no side effects